examples/plot_run_benchmark_python_R.py

```diff
@@ -34,5 +34,5 @@
 
 kinds = list(PLOT_KINDS.keys())
 figs = plot_benchmark(save_file, benchmark=Benchmark(BENCHMARK_PATH),
-                      kinds=kinds, html=False)
+                      kinds=kinds, html=False, reset_style=True)
```

Comment:
Actually, I am not sure I am in favor of adding such argument to the `plot_benchmark` function. Why not call explicitely `reset_solver_idx` here?

Reply:
Done. It seems way better indeed